AEPServices/Sources/ui/floating/FloatingButtonPresentable.swift

```diff
@@ -23,4 +23,11 @@ public protocol FloatingButtonPresentable: Showable, Dismissible {
     /// - Parameters:
     ///     - imageData : The `Data` representation of a UIImage
     func setButtonImage(imageData: Data)
+
+    /// Set the initial position of floating button.
+    /// By default the initial position is set to `FloatingButtonPosition.center`.
+    /// Call this method before calling `floatingButton.show()` to set the position of the floating button when it appears.
+    /// - Parameters:
+    ///     - position : The `FloatingButtonPosition` defining the initial position of the floating button.
+    func setInitialPosition(position: FloatingButtonPosition)
```

Comment:
Nit: I think we should name this API `setInitialPosition(position` to `setInitial(position` to allow the argument label to finish the name of the API. This is somewhat of a standard in Swift.

So the call would be like `setInitial(position: ...)` instead of `setInitialPosition(position: ...)`. Just slightly less repeteative.